src/compiler/es-next/hybrid-function.js

```diff
@@ -122,8 +122,7 @@ function compile (fnCode, dependenciesCode, createRegeneratorInClientCodeError)
     if (NODE_VER >= 4)
         fnCode = downgradeES(fnCode);
 
-    if (!hammerhead.isScriptProcessed(fnCode))
-        fnCode = hammerhead.processScript(fnCode, false);
+    fnCode = hammerhead.processScript(fnCode, false);
```

Comment:
Yes, we don't have functions as return value for hybrids. So there is no way compiled code can get here anymore